.storybook/themes/news.scss

```diff
@@ -633,6 +633,46 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				share-bar: (
+					box-shadow: (
+						0 4px 8px 0 rgba(0, 0, 0, 0.2),
+						0 6px 20px 0 rgba(0, 0, 0, 0.19),
+					),
+					border-radius: 0 5px 5px 0,
+					display: flex,
```

Comment:
```suggestion
```

Reply:
sounds like at one breakpoint there should be display none

Reply:
@JackHowa : Thank you for the commit suggestion. I did change the `display` to `none` for `default()` in my changes.